proxysql/README.md

```diff
@@ -14,6 +14,11 @@ The ProxySQL integration is included in the [Datadog Agent][3] package, so you d
 
 ### Configuration
 
+#### Enabling SSL
+To connect to ProxySQL using SSL/TLS, enable the `use_tls` option in `conf.yaml`. 
+
+Note: You should include any certificates and passwords needed to connect via SSL/TLS.
```

Comment:
```suggestion
To connect to ProxySQL using SSL/TLS, enable the `use_tls` option in `conf.yaml`. Include certificates and passwords needed to connect via SSL/TLS.
```
Users would probably find it helpful to have an example codeblock here, showing what this would look like, with `use_tls` enabled and example certs and passwords included.